app/uk/gov/hmrc/servicedependencies/service/DependencyGraphParser.scala

```diff
@@ -96,14 +96,14 @@ object DependencyGraphParser {
 
     def pathToRoot(node: Node): Seq[Node] = {
       @tailrec
-      def go(node: Node, acc: Seq[Node]): Seq[Node] = {
+      def go(node: Node, acc: Seq[Node], seen: Set[Node]): Seq[Node] = {
         val acc2 = acc :+ node
```

Comment:
or add to `acc` first to avoid `seen`?
```scala
def go(node: Node, acc: Seq[Node]): Seq[Node] = {
  arrowsMap.get(node) match {
    case Some(n) if !acc.contains(n) => go(n, acc :+ n)
    case _    => acc
  }
go(node, Seq(node))
```